### PR TITLE
fix(parsing): json parsing fixed for commented json files

### PR DESCRIPTION
**Pull request by yigitkurtcu**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?
Comments in json files were getting deleted on every parse.

Issue Number: [#2056](https://github.com/nestjs/nest-cli/issues/2056)

## What is the new behavior?
A new package added to parse and stringify JSON files with comments.

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by micalevisk — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "@angular-devkit/core": "16.0.1",
     "@angular-devkit/schematics": "16.0.1",
+    "comment-json": "^4.2.3",
```
Comment (kamilmysliwiec) [inline package.json:46]:
```suggestion
    "comment-json": "4.2.3",
```

File: package.json [hunk 2]
```diff
@@ -43,18 +43,19 @@
...
     "@types/jest": "29.5.1",
-    "@types/node": "18.16.14",
-    "@typescript-eslint/eslint-plugin": "5.59.7",
-    "@typescript-eslint/parser": "5.59.7",
+    "@types/node": "18.16.12",
+    "@typescript-eslint/eslint-plugin": "5.59.6",
+    "@typescript-eslint/parser": "5.59.6",
     "cpx": "1.5.0",
-    "eslint": "8.41.0",
+    "eslint": "8.40.0",
```
Comment (kamilmysliwiec) [inline package.json:58]: Can you revert these changes?

Comment (yigitkurtcu) [reply package.json]: Fixed 👍
